spec/models/call_log_spec.rb

```diff
@@ -1,7 +1,8 @@
 require 'rails_helper'
 
 RSpec.describe CallLog, type: :model do
-  describe 'Associations' do
-    it { should belong_to(:user) }
+  describe 'Validations' do
```

Comment:
I think this is unrelated to this PR?

Reply:
@timcheadle You are correct - this is unrelated, but these specs were failing so fixed this in the same PR.

Reply:
@timcheadle As discussed on Slack, moved out the specs changes into a a separate PR, and keeping only relevant changes in this PR. 👍